### PR TITLE
Members Cover Endpoints

### LINKED ISSUE #248: Member's cover endpoint
Building this PR #247 I've noticed there wasn't a cover endpoint for the user. Could you work on it @renatonascalves ?

### PR DESCRIPTION
**Pull request by renatonascalves**
fixes #248

### CODE REVIEW

File: tests/attachments/test-member-cover-controller.php [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+	public function test_get_item_with_no_image() {
+		$request  = new WP_REST_Request( 'GET', sprintf( $this->endpoint_url . '%d/cover', $this->user_id ) );
+		$response = $this->server->dispatch( $request );
+		$this->assertErrorResponse( 'bp_rest_attachments_user_cover_no_image', $response, 500 );
+	}
+
+	/**
+	 * @group get_item
+	 */
+	public function test_get_item_invalid_group_id() {
```
Comment (imath) [inline tests/attachments/test-member-cover-controller.php]: This function should probably be renamed to ` test_get_item_invalid_member_id()`

File: tests/attachments/test-member-cover-controller.php [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+		$request  = new WP_REST_Request( 'DELETE', sprintf( $this->endpoint_url . '%d/cover', $this->user_id ) );
+		$response = $this->server->dispatch( $request );
+		$this->assertErrorResponse( 'bp_rest_authorization_required', $response, rest_authorization_required_code() );
+	}
+
+	/**
+	 * @group delete_item
+	 */
+	public function test_delete_item_invalid_group_id() {
```
Comment (imath) [inline tests/attachments/test-member-cover-controller.php]: This function should probably be renamed to `test_delete_item_invalid_member_id()`

File: tests/attachments/test-member-cover-controller.php [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+
+		$request  = new WP_REST_Request( 'POST', sprintf( $this->endpoint_url . '%d/cover', $this->user_id ) );
+		$response = $this->server->dispatch( $request );
+		$this->assertErrorResponse( 'bp_rest_authorization_required', $response, rest_authorization_required_code() );
+	}
+
+	/**
+	 * @group create_item
+	 */
+	public function test_create_item_invalid_group_id() {
```
Comment (imath) [inline tests/attachments/test-member-cover-controller.php]: This function should probably be renamed to `test_create_item_invalid_member_id()`

**Comment by renatonascalves** [comment]
@imath Tks! Updated! And yes, `bp_cover_image_upload` is required for cover images upload for both groups and members. :)

**Comment by renatonascalves** [comment]
I'll take care of the `create_item` soon. I had some problems before and have been procastinating ever since to actually build it. :/

**Pull request merged**